src/celeritas/em/model/EPlusGGModel.cu

```diff
@@ -0,0 +1,36 @@
+//---------------------------------*-CUDA-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/em/model/EPlusGGModel.cu
+//---------------------------------------------------------------------------//
+#include "EPlusGGModel.hh"
+
+#include "celeritas/em/executor/EPlusGGExecutor.hh"
+#include "celeritas/global/ActionLauncher.device.hh"
+#include "celeritas/global/CoreParams.hh"
+#include "celeritas/global/CoreState.hh"
+#include "celeritas/global/TrackExecutor.hh"
+#include "celeritas/phys/InteractionApplier.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Interact with device data.
+ */
+void EPlusGGModel::execute(CoreParams const& params,
+                           CoreStateDevice& state) const
+{
+    auto execute = make_action_track_executor(
+        params.ptr<MemSpace::native>(),
+        state.ptr(),
+        this->action_id(),
+        InteractionApplier{EPlusGGExecutor{this->host_ref()}});
```

Comment:
Should be `this->device_ref()`. The use of `host_ref` in the `.cu` file I wrote was a mistake, as we discussed with Philippe. It returns the same data type for now so probably shouldn't matter?